content/en/network_performance_monitoring/installation.md

```diff
@@ -79,9 +79,47 @@ To enable network performance monitoring with the Datadog Agent, use the followi
 
     **Note**: If the `systemctl` command is not available on your system, run the following command instead: `sudo service datadog-agent restart`
 
+### SELinux-enabled systems
+
+On systems with SELinux enabled, the system-probe binary needs special permissions to use eBPF features.
+
+The Datadog Agent RPM package for CentOS-based systems bundles [an SELinux policy][3] to grant these permissions to the system-probe binary.
+
+If you need to use Network Performance Monitoring on other systems with SELinux enabled, do the following:
+
+1. Modify the base [SELinux policy][3] to match your SELinux configuration.
+    Depending on your system, some types or attributes may not exist (or have different names).
```

Comment:
Should we make it clearer a) how to do this b) how to know when this is required?